sources/packages/backend/apps/api/src/route-controllers/user/user.controller.service.ts

```diff
@@ -0,0 +1,53 @@
+import { UnprocessableEntityException, Injectable } from "@nestjs/common";
+import { IUserToken } from "../../auth/userToken.interface";
+import { UserService, BCeIDService } from "../../services";
+import { BCeIDAccountTypeCodes } from "../../services/bceid/bceid.models";
+import { SearchAccountOptions } from "../../services/bceid/search-bceid.model";
+import { BCeIDAccountsAPIOutDTO } from "./models/bceid-accounts.dto";
+
+@Injectable()
+export class UserControllerService {
+  constructor(
+    private readonly service: UserService,
+    private readonly bceidService: BCeIDService,
+  ) {}
+
+  async getAllBCeIDs(userToken: IUserToken): Promise<BCeIDAccountsAPIOutDTO> {
+    // Only business BCeID will execute a search on BCeID Web Services.
+    // Basic BCeID need to have the full user name provided to execute the
+    // getAccountDetail method on BCeID Web Services.
+    const account = await this.bceidService.getAccountDetails(
+      userToken.idp_user_name,
+      BCeIDAccountTypeCodes.Business,
+    );
+
+    if (!account) {
+      throw new UnprocessableEntityException(
```

Comment:
Maybe not part of the API reorganization, but this method was supposed to throw a `CustomNamedError` to them be handled in the controller with the proper HTTP exception.

Reply:
I can do it in another PR.